ghost/core/core/frontend/src/ghost-stats/ghost-stats.js

```
@@ -0,0 +1,297 @@
+  /**
+   * Send event to endpoint
+   *
+   * @param  { string } name Event name
+   * @param  { object } payload Event payload
+   * @return { Promise<any> } request response
+   */
+  async function _sendEvent(name, payload) {
+    _setSessionId()
+    const url = `${host}?name=${DATASOURCE}&token=${token}`
+
+    let processedPayload
```

ESLint

[error] Expected indentation of 8 spaces but found 4.

(indent)